gslib/utils/boto_util.py

```diff
@@ -198,11 +205,11 @@ def GetGcsJsonApiVersion():
 # in multiples of 256KiB). Overridable for testing.
 def GetJsonResumableChunkSize():
   chunk_size = config.getint('GSUtil', 'json_resumable_chunk_size',
-                             1024*1024*100L)
+                             long(1024*1024*100))
```

Comment:
AFAIK long is not even needed here (or in fact anywhere else)
1024*1024*100L is below 2147483647, the upper limit of int on 32bit python 2
also
https://www.python.org/dev/peps/pep-0237/
means that if it exceeds the limit it will be changed to long automatically

Reply:
We can probably make a TODO to drop `long` completely once Python 2 reaches end of life and this only runs under Python 3 (https://pythonclock.org/).  In the interim, I'd rather not go changing things without a good reason.